### ISSUE TITLE
RTD - commits won't trigger new builds

### ISSUE THREAD

**Issue by scls19fr** [python-semver/python-semver#95]
Hello @k-bx ,

python-semver doc is available at ReadTheDocs (RTD)
See https://python-semver.readthedocs.io/en/latest/

Although

https://readthedocs.org/projects/python-semver/ displays

```
This repository doesn't have a valid webhook set up, commits won't trigger new builds for this project.  See your project integrations for more information.
```

Could you create an account at https://readthedocs.org/
so I could add you as a maintainer also.

You will have to setup webhook according
http://docs.readthedocs.io/en/latest/webhooks.html
in this GitHub repository to be able to automatically build doc

Kind regards

**Comment by tomschr** [comment]
@scls19fr 
Maybe related to this is PR #127 if you want to build your documentation via tox.

**Comment by k-bx** [comment]
Hey. My account is k_bx there. Thanks

**Comment by scls19fr** [comment]
Hi @k-bx I've just add you as maintainer of [python-semver RTD](https://readthedocs.org/projects/python-semver/)

**Comment by scls19fr** [comment]
I've just triggered doc build manually 
to reflect updates from in @tomschr #143
see https://readthedocs.org/projects/python-semver/builds/9735766/
but doc should be updated without triggering manually (just when merging PR)

**Comment by tomschr** [comment]
If I did everything right, I've removed the old Webhook, added a new one and "reconnected" it to ReadTheDocs.

From my perspective it should be fixed. Whenever we add new commits to master, the Webhook will be triggered and create a new documentation update. The result will be available under the "latest" link.

Maybe someone can confirm? If nobody objects, I'll take the freedom to close this issue tomorrow as fixed. :wink:
